eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -182,43 +182,46 @@ class EclairImpl(appKit: Kit) extends Eclair {
   }
 
   override def audit(from_opt: Option[Long], to_opt: Option[Long])(implicit timeout: Timeout): Future[AuditResponse] = {
-    val from = from_opt.getOrElse(0L)
-    val to = to_opt.getOrElse(MaxEpochSeconds)
+    val filter = getDefaultTimestampFilters(from_opt, to_opt)
 
     Future(AuditResponse(
-      sent = appKit.nodeParams.db.audit.listSent(from, to),
-      received = appKit.nodeParams.db.audit.listReceived(from, to),
-      relayed = appKit.nodeParams.db.audit.listRelayed(from, to)
+      sent = appKit.nodeParams.db.audit.listSent(filter.from, filter.to),
+      received = appKit.nodeParams.db.audit.listReceived(filter.from, filter.to),
+      relayed = appKit.nodeParams.db.audit.listRelayed(filter.from, filter.to)
     ))
   }
 
   override def networkFees(from_opt: Option[Long], to_opt: Option[Long])(implicit timeout: Timeout): Future[Seq[NetworkFee]] = {
-    val from = from_opt.getOrElse(0L)
-    val to = to_opt.getOrElse(MaxEpochSeconds)
+    val filter = getDefaultTimestampFilters(from_opt, to_opt)
 
-    Future(appKit.nodeParams.db.audit.listNetworkFees(from, to))
+    Future(appKit.nodeParams.db.audit.listNetworkFees(filter.from, filter.to))
   }
 
   override def channelStats()(implicit timeout: Timeout): Future[Seq[Stats]] = Future(appKit.nodeParams.db.audit.stats)
 
   override def allInvoices(from_opt: Option[Long], to_opt: Option[Long])(implicit timeout: Timeout): Future[Seq[PaymentRequest]] = Future {
-    val from = from_opt.getOrElse(0L)
-    val to = to_opt.getOrElse(MaxEpochSeconds)
+    val filter = getDefaultTimestampFilters(from_opt, to_opt)
 
-    appKit.nodeParams.db.payments.listPaymentRequests(from, to)
+    appKit.nodeParams.db.payments.listPaymentRequests(filter.from, filter.to)
   }
 
   override def pendingInvoices(from_opt: Option[Long], to_opt: Option[Long])(implicit timeout: Timeout): Future[Seq[PaymentRequest]] = Future {
-    val from = from_opt.getOrElse(0L)
-    val to = to_opt.getOrElse(MaxEpochSeconds)
+    val filter = getDefaultTimestampFilters(from_opt, to_opt)
 
-    appKit.nodeParams.db.payments.listPendingPaymentRequests(from, to)
+    appKit.nodeParams.db.payments.listPendingPaymentRequests(filter.from, filter.to)
   }
 
   override def getInvoice(paymentHash: ByteVector32)(implicit timeout: Timeout): Future[Option[PaymentRequest]] = Future {
     appKit.nodeParams.db.payments.getPaymentRequest(paymentHash)
   }
 
+  def getDefaultTimestampFilters(from_opt: Option[Long], to_opt: Option[Long]) = {
```

Comment:
Maybe make that a method of `TimestampQueryFilter` companion object?

Reply:
In 409a7fa I moved the function to the companion object and grouped together the case classes.